lib/reversetunnel/srv.go

```diff
@@ -402,7 +402,7 @@ func (s *server) diffConns(newConns, existingConns map[string]services.TunnelCon
 }
 
 func (s *server) Wait() {
-	s.srv.Wait()
+	s.srv.Wait(context.TODO())
```

Comment:
Would it be better to have a caller pass context to the outer method (similar to Shutdown below)? Or is it too much refactoring?

Reply:
yeah, no one is using it now, so I decided to pass